### PR TITLE
Added sv.yml

### PR DESCRIPTION
**Pull request by Ungutvecklare**
Added some swedish translations. Changes will come later.

### CODE REVIEW

**Comment by Ungutvecklare** [comment]
I think i put swe instead of sv in the YML file, dont know if its gonna cause errors.

**Comment by 715209** [comment]
Hi [Ungutvecklare](https://github.com/Ungutvecklare),

Thank you for the translation! The file looks good.

File: locales/sv.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+swe:
+    start:
+        success: Strömmen startades framgångsrikt
+        error: "Fel kan inte starta strömmen: %{error}"
+    startTwitchTranscoding:
+        trying: Försöker starta strömmen med omkodning
+        successNoTranscoding: Strömmen startades framgångsrikt utan omkodning
+        success: Startade ström med omkodning %{attemptsMessage}
+        attempts: ", took %{count} attempts"
```
Comment (715209) [inline locales/sv.yml:9]: Could you please translate this as well.

File: locales/sv.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+        errorAlreadyUsed: "%{alias} används redan som alias"
+        errorIncorrectArguments: Fel felaktiga argument har angetts
+    sceneSwitch:
+        switch: Scenen byttes till "%{scene}"
+    handleCommands:
+        notify: Automatisk anmälan om växling %{condition}
+        autostop: Automatiskt stopp för strömmen %{condition}
+        enabled: är aktiverad
+        disabled: är inaktiverat
+    noalbs:
```
Comment (715209) [inline locales/sv.yml:51]: This can also be translated, doesn't have to stay in English.

File: locales/sv.yml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+        errorPrefix: Can't update NOALBS prefix
+        switcherEnabled: Successfully enabled the switcher
+        switcherDisabled: Successfully disabled the switcher
+        instantSwitch: Instant switch on recover %{condition}
+        langError: Can't update NOALBS language
+        langErrorInvalid: Error editing language %{lang} is not a valid value
+        langSuccess: NOALBS language updated to %{lang}
+        retryCount: Current retry set at %{count}
+        retryError: Error editing retry attempts %{count} is not a valid value
+        retrySuccess: Retry attempts set to %{count}
```
Comment (715209) [inline locales/sv.yml:62]: This can also be translated, doesn't have to stay in English.

File: locales/sv.yml (hunk 1 above)
Comment (Ungutvecklare) [reply locales/sv.yml:9]: > Could you please translate this as well.

Some of it is not directly translatable but ill get on it :)

File: locales/sv.yml (hunk 3 above)
Comment (Ungutvecklare) [reply locales/sv.yml:62]: Some of it is not directly translatable but ill get on it :)

**Review by Ungutvecklare — COMMENTED** [review]
Requested corrections has been made to the swedish translations.

**Comment by 715209** [comment]
Hi [Ungutvecklare](https://github.com/Ungutvecklare),

The file looks good to me! Thank you for the translation, it will be available in the next release.

**Pull request merged**